Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && apt-get install -y \
+    pkg-config \
+    libpng-dev \
+    libjpeg8-dev \
+    libfreetype6-dev \
+    libblas-dev \
+    liblapack-dev \
+    libatlas-base-dev \
+    gfortran \
+    python \
+    python-dev \
+    python-pip \
+    curl && \
+    curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash - && \
+    apt-get install -y nodejs
+
+RUN npm install webpack --save-dev
+RUN npm install webpack -g
```

Comment:
You have to have webpack installed globally in order to use the CLI tool. Maybe that's why? The Front End instructions in README.md already tells you to install those separately. I'm not sure if they are necessary to install in the Dockerfile if you install them manually